Adyen.Test/ByteArrayToStringTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Adyen.Model;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Adyen.Test
+{
+    [TestClass]
+
+    public class ByteArrayToStringTest
+    {
+        [TestMethod]
+        public void TestSerialize()
+        {
+            var json = "{\"cavv\":\"Qnl0ZXMtVG8tQmUtRW5jb2RlZA==\",\"xid\":\"Qnl0ZXMtVG8tQmUtRW5jb2RlZA==\"}";
+            var bytesArray = Convert.FromBase64String("Qnl0ZXMtVG8tQmUtRW5jb2RlZA==");
+            var threeDSecure = new ThreeDSecureData
+            {
+                Cavv = bytesArray,
```

Comment:
should this not be the base64 value? You know use the actual value not the base64 value